### PR TITLE
Kel hotel backend

### PR DESCRIPTION
**Pull request by keldim**

### CODE REVIEW

File: backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/service/HotelService.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@Service
+@RequiredArgsConstructor
+public class HotelService {
+    private final HotelRepository repository;
+
+    public List<Hotel> findAll() {
+        return repository.findAll();
+    }
+
+    public Hotel findById(long id) {
```
Comment (shawnkoong) [inline backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/service/HotelService.java:20]: i think it is cleaner to have something like
`Hotel hotel = repository.findById(id).orElseThrow()`
and also throwing something more appropriate like entity not found exception

**Review by shawnkoong — APPROVED** [review]
Approve for now.

Are you going to add functionality to add/save hotel as part of a trip in this PR or another one?

File: backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/service/HotelService.java (hunk 1 above)
Comment (keldim) [reply backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/service/HotelService.java:20]: you are right, I'll change it

**Comment by keldim** [comment]
> Approve for now.
> 
> Are you going to add functionality to add/save hotel as part of a trip in this PR or another one?

I'll create another PR for that functionality

**Pull request merged**